functionnal_tests/cypress/integration/profile/edit_react_json_schema_form.js

```diff
@@ -0,0 +1,57 @@
+import { PAGES } from '../../support/urls_commands'
+import baseWorkspace from '../../fixtures/baseWorkspace.json'
+import baseOtherUser from '../../fixtures/baseOtherUser.json'
+import defaultAdmin from '../../fixtures/defaultAdmin.json'
+
+describe("Editing the user profile", () => {
+  before(function () {
+    cy.resetDB()
+    cy.setupBaseDB()
+    cy.login(defaultAdmin)
+    cy.visitPage({ pageName: PAGES.PROFILE, params: { userId: defaultAdmin.user_id } })
+  })
+
+  describe('Information form', () => {
+    describe('while receiving a TLM', () => {
+      it('should not have reset the value of the first input', () => {
+        cy.get(`[data-cy=CustomFormManager__updateProfile__edit__button]`)
+          .first()
+          .should('be.visible')
+          .click()
+
+        const expectedText = 'some text'
+        cy.get('form.rjsf .form-group > input[type="text"]')
+          .first()
+          .type(expectedText)
+
+        cy.createUser('baseOtherUser').then(user => {
+          cy.addUserToWorkspace(user.user_id, baseWorkspace.workspace_id)
+
+          cy.clearCookies() // CH - INFO - 20210128 - clearing cookies to force use basic auth to create the content with baseOtherUser
+
+          cy.request({
+            method: 'POST',
+            url: `/api/workspaces/${baseWorkspace.workspace_id}/contents`,
+            auth: {
+              user: baseOtherUser.email,
+              pass: baseOtherUser.password
+            },
+            body: {
+              content_type: 'html-document',
+              label: 'some note'
+            }
+          }).then(() => {
```

Comment:
I think the `then()` is not needed here as `cy.` commands are always executed sequentially, aren't they?

Reply:
I think your right, I will try

Reply:
without the .then, the test sometimes fails randomly in the `expect()` by testing `'s' === 'some text'` so I'm assuming it's important.

Won't fix